server/src/main/java/io/whitefox/core/ReadTableRequest.java

```diff
@@ -0,0 +1,154 @@
+package io.whitefox.core;
+
+import io.whitefox.annotations.SkipCoverageGenerated;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public interface ReadTableRequest {
```

Comment:
All of these ReadTableRequest doesn't have the boolean param "includeFiles".

Reply:
And also the maxFiles. Although at this moment it is not necessary, but what about adding a TODO?

Reply:
Same for starting and ending version and pageToken

Reply:
Addressed as issue #91 #92 #93 #94